### PR TITLE
ADEN-7942 Add tz_offset field to tracking.

### PR DESCRIPTION
**Pull request by jacekbj**

### CODE REVIEW

File: src/ad-engine/listeners/porvata-listener.js [hunk 1]
```diff
@@ -92,7 +92,8 @@ export class PorvataListener {
 			position: this.params.position ? this.params.position.toLowerCase() : '(none)',
 			// @DEPRECATED
 			browser: `${client.getOperatingSystem()} ${client.getBrowser()}`,
-			timestamp: new Date().getTime()
+			timestamp: new Date().getTime(),
+			tz_offset: new Date().getTimezoneOffset()
```
Comment (fraszczakszymon) [inline src/ad-engine/listeners/porvata-listener.js]: Can we reuse `new Date()`?

File: src/ad-engine/listeners/slot-listener.js [hunk 2]
```diff
@@ -43,6 +43,7 @@ function getData(adSlot, { adType, status }) {
 		page_width: window.document.body.scrollWidth || '',
 		time_bucket: (new Date()).getHours(),
 		timestamp: new Date().getTime(),
+		tz_offset: new Date().getTimezoneOffset(),
```
Comment (fraszczakszymon) [inline src/ad-engine/listeners/slot-listener.js]: Can we reuse `new Date()`?

Comment (jacekbj) [reply src/ad-engine/listeners/slot-listener.js]: Done

Comment (dmnsobczak) [inline src/ad-engine/listeners/slot-listener.js]: And what about those 3 `Dates`? :(

File: src/ad-products/tracking/video/video-event-data-provider.js [hunk 3]
```diff
@@ -40,6 +40,7 @@ export default {
 			price: '',
 			skin: context.get('targeting.skin'),
 			timestamp: now.getTime(),
+			tz_offset: new Date().getTimezoneOffset(),
```
Comment (dmnsobczak) [inline src/ad-products/tracking/video/video-event-data-provider.js]: Maybe `now.getTimezoneOffset(),`?

File: src/ad-engine/listeners/slot-listener.js (hunk 2 above)
Comment (jacekbj) [reply src/ad-engine/listeners/slot-listener.js]: Refactored.

File: src/ad-products/tracking/video/video-event-data-provider.js (hunk 3 above)
Comment (jacekbj) [reply src/ad-products/tracking/video/video-event-data-provider.js]: Done.

**Pull request merged**